data/monster/monsters.xml

```diff
@@ -93,7 +93,6 @@
 	<monster name="Boar" file="monsters/boar.xml" />
 	<monster name="Bog Frog" file="monsters/bog_frog.xml" />
 	<monster name="Bog Raider" file="monsters/bog_raider.xml" />
-	<monster name="Bone Beast" file="monsters/bonebeast.xml" />
```

Comment:
IIRC this was done on purpose as a lot of maps spawns had it types like this 🙈

Reply:
yeah they also had assasins and beholders, but we are not adjusting for errors (at least not in this case lol)